samples/misc/vaultGeneralRead.py

```diff
@@ -0,0 +1,58 @@
+import balpy
```

Comment:
I see what you've tried to here and I agree it makes it a bit easier to parse -- I'm just a bit wary about any feedback since they've already given us feedback about adhering to the current standards. Do you think it's worth sending it this way or should we preemptively update this to use the same `def main(): ...` standard we see in other scripts?

Reply:
I am not completely sure about what is the best approach here. We could definitely do it the way that is (adding one file for each vault function) but I think that this way is better to avoid having a lot of sample files.